package.json

```diff
@@ -15,7 +15,7 @@
     "postinstall": "./scripts/postInstall.sh",
     "clean": "./scripts/clean.sh",
     "clean-standalone": "STANDALONE_NEW_DOT=true ./scripts/clean.sh",
-    "android": "./scripts/set-pusher-suffix.sh && ./scripts/run-build.sh --android",
+    "android": "./scripts/set-pusher-suffix.sh && cd Mobile-Expensify && npm run grunt:build:shared && cd .. && ./scripts/run-build.sh --android",
```

Comment:
Don't we need to run this on iOS too?

Reply:
I'm not sure. I saw [this](https://github.com/Expensify/Mobile-Expensify/blob/262a4953a9f40be13b7d568c86418f71b96e203c/iOS/Expensify.xcodeproj/project.pbxproj#L3462-L3479) iOS build step which made me think no... and we only added the step to the Android workflows. Not 100% sure though

Reply:
I _think_ this should be running via grunt, but for whatever reason it's not working and engineers miss the error.

Reply:
https://github.com/Expensify/Mobile-Expensify/blob/main/Android/build.gradle#L405-L408

Reply:
Oooh that would be better. Perhaps our current grade command no longer matches [this](https://github.com/Expensify/Mobile-Expensify/blob/0029d3f85e87c1c84aa290d41bfe7c1861bebaca/Android/build.gradle#L385) logic: 

`theTask.name.matches("(assemble|bundle).*(Debug|Adhoc|Release)")`

Reply:
I think that cleanest solution would be to include the task to the gradle script. Now I think there is a chance that the `React Native CLI` may interfere into the build phases, or indeed change it in some way so the `buildGrunt` doesn't trigger.

Maybe changing the pattern to `(install|assemble|bundle).*(Debug|Adhoc|Release)` would be enough?

Reply:
Agree this is a much better fix. The suggestion appears to work from my testing: 

- Manual Generate Signed APK
    - <img width="1377" alt="Screenshot 2025-01-21 at 11 42 33" src="https://github.com/user-attachments/assets/386f5081-4145-4862-8366-0b4feb8f4253" />

- `npm run android`
    - <img width="978" alt="Screenshot 2025-01-21 at 11 51 32" src="https://github.com/user-attachments/assets/671336d4-ea46-469d-a666-1332f23eab92" />